demo/python/requirements.txt

```diff
@@ -1,4 +1,4 @@
 numpy
 pvporcupine==1.9.5
```

Comment:
pvporcupine==2.0.0

i know it's not published yet, but the demo is not gonna also work with the old one as well

Reply:
also increase the version in `setup.py` for the binding